Update LICENSE

anonymous19700101a opened the pull request:
Checks and balances on the U.S. Government are important. It is my belief that this pull request partially restores checks and balances on the U.S. Government.

<!--  Thanks for sending a pull request!  Here are some tips for you:

1. If this is your first time, please read our contributor guidelines: https://git.k8s.io/community/contributors/guide/first-contribution.md#your-first-contribution and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
2. Please label this pull request according to what type of issue you are addressing, especially if this is a release targeted pull request. For reference on required PR/issue labels, read here:
https://git.k8s.io/community/contributors/devel/sig-release/release.md#issuepr-kind-label
3. Ensure you have added or ran the appropriate tests for your PR: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
4. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
5. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
-->

#### What type of PR is this?

<!--
/kind feature
-->

#### What this PR does / why we need it:
This Pull Request restores Checks and Balances on the U.S. Government. The U.S. government has too much power over U.S. owned corporations, and this pull request should help address this issue. The specifics of the license changes should be discussed with civility.

#### Which issue(s) this PR fixes:
<!--
*Automatically closes linked issue when PR is merged.
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
_If PR is about `failing-tests or flakes`, please post the related issues/tests in a comment and do not use `Fixes`_*
-->
Fixes #

#### Special notes for your reviewer:
This needs review by someone smarter than myself. Please include discourse by the community.

#### Does this PR introduce a user-facing change?
<!--
If no, just write "NONE" in the release-note block below.
If yes, a release note is required:
Enter your extended release note in the block below. If the PR requires additional action from users switching to the new release, include the string "action required".

For more information on release notes see: https://git.k8s.io/community/contributors/guide/release-notes.md
-->
```release-note
This pull request prevents U.S. businesses from receiving security updates for Kubernetes unless a D.E.I. policy is in place.
In addition, it prevents U.S. businesses from using this software for developing A.I. systems to monitor the public, or develop A.I. weapons.
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

<!--
This section can be blank if this pull request does not require a release note.

When adding links which point to resources within git repositories, like
KEPs or supporting documentation, please reference a specific commit and avoid
linking directly to the master branch. This ensures that links reference a
specific point in time, rather than a document that may change over time.

See here for guidance on getting permanent links to files: https://help.github.com/en/articles/getting-permanent-links-to-files

Please use the following format for linking documentation:
- [KEP]: <link>
- [Usage]: <link>
- [Other doc]: <link>
-->
```docs

```

aojea commented:
is this the right place for a license change on the apache license?

pacoxu commented:
https://github.com/cncf/foundation/blob/main/charter.md#11-ip-policy CNCF charter has some general requirements as well.

Also you may refer to https://www.apache.org/foundation/license-faq.html#:~:text=by%20the%20ASF).-,May%20I%20re%2Duse%20(and%20modify)%20the%20Apache%20License,seek%20your%20own%20legal%20advice..

anonymous19700101a commented:
@aojea
For now, I believe so. As @pacoxu mentioned, it is possible to modify the Apache license. I believe due to the criticalness of events unfolding in the United States, it is important to set an example for major corporations as quickly as possible. I whole heartedly agree with you that the original Apache License should be updated, and that possibly different versions of the Apache License may be provided with different clauses. I think that will take too much time. Furthermore, if the Apache Software Foundation is compromised in anyway, I believe it to be more important to have an independent license here.

anonymous19700101a commented:
@pacoxu 
I have updated the pull request with new names for the license, and removed "Apache" from the License.

Per the CNCF Foundation Charter, it appears that this License change needs to be reviewed by a governing body. How might that review take place?

pacoxu commented:
Thank you for your proposal to modify the Kubernetes license. It's important to note that, as per the CNCF's IP Policy, any changes to the project's license require approval from the CNCF Governing Board. 

However, before reaching that stage, it's essential to engage in internal discussions within the Kubernetes community. 
- probably, it is more suitable to open an issue in https://github.com/kubernetes/community/issues to discuss  this matter further. Engaging in such discussions will help ensure that any proposed changes align with the community's goals and values.

aojea commented:
/hold

relicensing a project like kubernetes need a very deep and large discussion that can not happen in a PR, I suggest you to engage the the Kubernetes community and CNCF first, introducing your idea to the mailing lists, forums, governing boards, ...

aojea commented:
@anonymous19700101a with all respect I'm going to close this PR since as we explained is not the right process and place for this change. Licensing in oss is complex, licensing in a foundation is even more and relicensing is even more ... I found this article that you may find useful to understand more the context https://www.cncf.io/blog/2017/02/01/cncf-recommends-aslv2/ 

/close
